docs/it-preparation.md

```diff
@@ -0,0 +1,148 @@
+# IT Preparation
+
+
+## Contents
+
+ - [Definitions](#def)
+ - [Job Terms](#term)
+
+<br/><a name="def"></a>
+## Definitions
+
+ - 802.1X
+	- Enhance security of WLAN by IEEE, provides authentication frame work, allows users to be authenticated by a central authority. wireless.
+ - Authenticated DHCP
+	- First network access control, authenticating user id/password be for delivering a DHCP.
+ - Backbone
+	- Primary connectivity mechanism of a hierarchical distribution system. All systems that have connectivity to the backbone are assured of connectivity to each other. 
+ - Blacklisting
+ 	- An access control system that denies entry to specific users, programs, or net work addresses
+ - Berkeley Internet Name Domain (BIND)
+ 	- The most commonly used DNS service of the internet
+ - Broadcasting
+ 	- A packet that is received by all stations in the domain
+ - Cloud Computing
+ 	- The logical computational resources (data, software, hardware), accessible via a computer network. e.g. Gmail
+ - Command-line interface (CLI)
+ 	- It is a mechanism for interacting with an operating system. It contrasts with a graphical user interface (GUI), or menus on a text user interface (TUI). Many companies use this mechanism to communicate with their systems.
+ - Cyclic Redundant Check (CRC)
+ 	- A mathematical calculation on a frame work or cell that is used for error detection. If two CRCs don't match, there is an error.
+ - DDI
+ 	- a unified service or solution that integrate DNS, DHCP, and IPAM (IP Address Management) into one.
+ - Dynamic Host Configuration Protocol
+ 	- Assigning IP address to device
+ - Domain Name System
+ 	- the system of domain names. eg. google.com (no www) godaddy.com
+ - Frame
+ 	- A unit of data transmission in layer two, containing a packet to layer three
+ - File Transfer Protocol (FTP)
+ 	- the protocol to transfer files from one host to another. eg. cyberduck (assure the security of transfer). Now people use FTP. 
+ - Hypertext Transfer Protocol (HTTP)
+ 	- Protocol that supports request-response from a server. eg. a browser sends hyper text "www.google.com" to google.com through HTTP, then google.com returns a HTML to our browser
+ - Hop
+ 	- Each time a packet is forwarded, it undergoes a "hop". (traceroute www.google.com)
+ - IP Address Management (IPAM)
+ 	- The administration of DNC and DHCP. It means Planing, tracking, and managing the Internet Protocol space used in a network. eg. DNS knowing the IP address taken via DHCP, and updating itself.
+ - Local Area Network (LAN)
+ 	- Its a Network that connects computers and devices in a limited geographical area. oppose with WANs (Wide Area Network). eg. home and school. Smaller area, faster speed, no need for telecommunication line. 
+
+<br/><a name="terms"></a>
+## Job Terms
+
+### Hub, Switch and Router
+ - Hub
+	- connects all the network devices together
```

Comment:
Should not have so many spaces. And usually the first level uses "`*`" instead of "`-`"